bots/ec2_delete_key_pair.py

```diff
@@ -0,0 +1,30 @@
+'''
+## delete_key_pair
+What it does: Deletes a key pair
+Usage: AUTO: delete_key_pair
+Limitations: none
+'''
+import boto3
+from botocore.exceptions import ClientError
+
+### Delete EC2 Key Pair ###
+def run_action(boto_session, rule, entity, params):
+    #Get the key name
+    keypair = entity.get('id')
+
+    try:
+        #Delete the keypair
+        ec2_client = boto_session.client('ec2')
+        result = ec2_client.delete_key_pair(KeyName=keypair)
+        text_output = "Keypair %s was deleted successfully\n" % keypair
```

Comment:
please use this pattern to print:
text_output = f'Keypair {keypair} was deleted successfully.\n'